roles/syslog-aggregator/tasks/main.yml

```diff
@@ -24,7 +24,7 @@
   cron:
     name: log-cleanup
     user: root
-    job: "find {{ log_root }} -mtime +28 -type f -exec ls {} \\;"
+    job: "find {{ log_root }} -mtime +28 -type f -exec rm {} \\;"
```

Comment:
I don't know; probably both. With `;` find executes the program for each argument while with `+` it uses as many arguments as possible for each execution. I've done some operations (I *think* not rm) with `;` that took hours but using `+` finished in seconds.